src/main/java/com/exactpro/th2/common/event/EventUtils.java

```diff
@@ -36,12 +36,13 @@ public static Message createMessageBean(String text) {
         return new MessageBuilder().text(text).build();
     }
 
-    @Contract("null -> null; !null -> !null")
-    public static @Nullable EventID toEventID(@Nullable String id) {
+    @Contract("_, null -> null; _, !null -> !null")
+    public static @Nullable EventID toEventID(String bookName, @Nullable String id) {
```

Comment:
This is part of a public API please add a separate override method.

Reply:
Discussed in chat